docs/en/compute/HBase.md

```diff
@@ -69,12 +69,15 @@ Alternative ways are described in the [Advanced Setup]({{ '/en/compute/HBase.htm
 
 ## Example
 
-Start HBase:
+Ensure alluxio scheme is recognized before starting HBase:
 
 ```bash
 ${HBASE_HOME}/bin/start-hbase.sh
 ```
 
+If not, follow the [Usage FAQs](https://docs.alluxio.io/os/user/1.4/en/Debugging-Guide.html#q-why-do-i-see-exceptions-like-no-filesystem-for-scheme-alluxio)
```

Comment:
instead of 1.4 use the variable `{{site.ALLUXIO_MAJOR_VERSION}}`

Reply:
updated, thanks!!